sdk/file_sdk/UnitTests/xmp_ut/xmp_ut.pro

```diff
@@ -22,13 +22,14 @@ win32:LIBS += -L$$REPO_ROOT/third_party/lib/xmp
 
 CONFIG(debug, debug|release) {
     TARGET = $$join(TARGET,,,d)
-    LIBS +=  -lrmscryptod -lrmsd -lmodxmpfiled -lmodfilecommond -lXMPCoreStaticD -lXMPFilesStaticD
+    LIBS +=  -lrmscryptod -lrmsd -lmodxmpfiled -lmodfilecommond
+    win32:LIBS += -lXMPCoreStaticD -lXMPFilesStaticD
 } else {
     LIBS +=  -lrmscrypto -lrms -lmodxmpfile -lmodfilecommon
     win32:LIBS += -lXMPCoreStatic -lXMPFilesStatic
 }
 
-unix:!mac:LIBS += -lstaticXMPFiles -lstaticXMPCore
+unix:!mac:LIBS += -L/usr/lib/xmp -lXMPCore -lXMPFiles -ldl

```

Comment:
>unix:!mac:LIBS += -L/usr/lib/xmp [](start = 0, length = 32)

shouldn't be. use linux default (/usr/lib) and explicitly include <xmp/...> for simplicity

Reply:
No, it shouldn't. its not include folder, its a lib folder.